MBBSEmu/HostProcess/MbbsHost.cs

```diff
@@ -857,22 +857,18 @@ private void ProcessIncomingCharacter(SessionBase session)
                         if (!session.TransparentMode && session.CharacterProcessed > 0)
                             session.SendToClient(new byte[] { 0xD, 0xA });
 
-                        //If there's no status in the queue, we can bail here
-                        if (session.Status.Count == 0)
-                        {
-                            session.Status.Enqueue(3);
-                            break;
-                        }
-
                         //If BTUCHI Injected a deferred Execution Status, respect that vs. processing the input
                         if (session.GetStatus() == 240)
                         {
                             //Set Status == 3, which means there is a Command Ready
                             session.Status.Clear(); //Clear the 240
                             session.Status.Enqueue(3); //Enqueue Status of 3
                             session.EchoSecureEnabled = false;
+                            break;
                         }
 
+                        //Always Enqueue Input Ready
```

Comment:
would changing the default to 3 do the same thing?

https://github.com/mbbsemu/MBBSEmu/blob/3257d2cc8390077d8e8e69949b3e43d6b99b95de/MBBSEmu/Session/SessionBase.cs#L339

Reply:
No -- if we defaulted the status to 3, it'd assume there was input to process on each iteration of the event loop and trigger `STTROU`. We default it to 1 (OK/READY) which lets us know there's nothing that needs to be done.